src/ProxySettings.cpp

```diff
@@ -9,7 +9,11 @@ namespace aws { namespace iot { namespace securedtunneling { namespace settings
     using boost::property_tree::ptree;
     
     char const * const KEY_DEFAULT_BIND_ADDRESS = "tunneling.proxy.default_bind_address";
+#ifdef _ALL_IPV4_ADDRESSES
+    std::string DEFAULT_DEFAULT_BIND_ADDRESS = "0.0.0.0";
```

Comment:
Is there a common use case for changing the binding IP addresses? If it's just either localhost or 0.0.0.0 then it's not necessary to add a CLI argument for it, but if there are other use cases for binding to different interfaces, then it would make sense. Using a build flag just make sure the binary is changed only for the docker use case.

Reply:
I am referring to an existing CLI parameter [here](https://github.com/aws-samples/aws-iot-securetunneling-localproxy/blob/892bd4af0a010abe075e224610daf63d656b02d0/src/main.cpp#L149) .
So I am wondering if you introduce this flag because this parameter is not working or can not suite the use case here.

Reply:
Oh, I see, yeah that parameter can be used, only difference would be that the user would need to pass that flag when running localproxy from within a  container, I'll remove that build definition and update the instructions instead.